mods/doors/init.lua

```diff
@@ -452,6 +452,50 @@ doors.register("door_obsidian_glass", {
 		},
 })
 
+-- Capture mods using the old API as best as possible.
+function doors.register_door(name, def)
+	if def.only_placer_can_open then
+		def.protected = true
+	end
+	def.only_placer_can_open = nil
+	local i = name:find(":")
+	local modname = name:sub(1, i - 1)
+	local doorname = modname .. "_" .. name:sub(i + 1, -1)
```

Comment:
Please take the habit of separating a list of variables from the rest with spaces. One variable is fine, above that -> you isolate them.